src/preferences/types.ts

```diff
@@ -24,6 +24,9 @@ export type UserPreferences = {
   showTestIds?: boolean;
   hideProductDescriptions?: boolean;
   debugShowSeconds?: boolean;
+  // A toggle inside the accessibility settings: it only disables the travel aid feature in the app,
+  // not the entire accessibility settings
+  travelAid?: boolean;
```

Comment:
We should make naming consistent here it a separate PR ("journey aid" seems to be correct), but let's make the local storage stuff correct from the start:

```suggestion
  journeyAidEnabled?: boolean;
```